projects/plugins/crm/modules/mailpoet/includes/class-mailpoet-background-sync-job.php

```diff
@@ -621,25 +613,18 @@ public function percentage_completed( $return_counts = false, $use_cache = true
 			$percentage_completed = 100;
 
 		} else {
-
-			$percentage_completed = $this->current_page / $this->mailpoet_total_pages * 100;
-
+			$percentage_completed = 100 * ( $this->current_page + 1 ) / $this->mailpoet_total_pages;
 		}
 
 		$this->debug( 'Percentage completed: ' . $percentage_completed . '%' );
-
-		$this->debug( 'Pages completed: ' . $this->current_page . ' / ' . $this->mailpoet_total_pages );
-		$this->debug( 'Subscribers completed: ' . min( $this->current_page * $this->subscribers_per_page, $this->mailpoet_total_subscribers ) . ' / ' . $this->mailpoet_total_subscribers );
-		$this->debug( 'Percentage completed: ' . $percentage_completed . '%' );
+		$this->debug( 'Pages completed: ' . ( $this->current_page + 1 ) . ' / ' . $this->mailpoet_total_pages );
```

Comment:
We have to match ranges, so the current page should be increase +1 for this debug info.